rare/components/tabs/games/game_list.py

```diff
@@ -1,12 +1,14 @@
 import os
 from logging import getLogger
 
+import gi
```

Comment:
What is this? I don't see it used anywhere...

Reply:
Other than this it works great!

Reply:
> What is this? I don't see it used anywhere...

Idk??



> Other than this it works great!

Nice!